lib/src/command/outdated.dart

```diff
@@ -1093,3 +1102,29 @@ class _FormattedString {
 
   static String _noFormat(String x) => x;
 }
+
+/// Whether the package [name] is overridden anywhere in the workspace rooted at
+/// [workspaceRoot].
+bool hasOverride(Package workspaceRoot, String name) {
+  return workspaceRoot.allOverridesInWorkspace.containsKey(name);
+}
+
+/// Whether the package [name] is depended on directly anywhere in the workspace
+/// rooted at [workspaceRoot].
+bool hasDependency(Package workspaceRoot, String name) {
+  return workspaceRoot.transitiveWorkspace
+      .any((p) => p.dependencies.containsKey(name));
+}
+
+/// Whether the package [name] is dev-depended on directly anywhere in the workspace
+/// rooted at [workspaceRoot].
+bool hasDevDependency(Package workspaceRoot, String name) {
+  return workspaceRoot.transitiveWorkspace
+      .any((p) => p.devDependencies.containsKey(name));
+}
```

Comment:
nit: these would be neat as extension methods?

Reply:
I think I'll refactor these in a follow-up